libraries/libfc/include/fc/container/ordered_diff.hpp

```diff
@@ -52,6 +63,7 @@ class ordered_diff {
                   result.insert_indexes.emplace_back(t, target[t]);
                   ++s;
                   ++t;
+                  assert(s > 0 && t > 0);
```

Comment:
considering that `s` and `t `are of type `size_t`, I don't think these asserts are necessary!

Reply:
Makes sense, I'll remove them.